### PR TITLE
Space Ruin: Rework Syndie Research Base

### PR DESCRIPTION
**Pull request by Venuska1117**
## What Does This PR Do
Changes whole syndicate base ground up, support now up to 8 researchers, contain now toxin test arena and its protected by ancient stone + turrets and expand all other stuff.

## Why It's Good For The Game
make syndie base more protected against explorers "raiding it", there was discussion about it on discord and they said that syndie base shuld not be raided.

now it support up to 8 researchers because base is bigger now, added a 8 new areas for syndicate base and now full protection from turrets and syndicate ed209 (admin activated) and hidden stash of 2000 credits for syndicate use ~~for NT merchandise vending machine,~~ to help learn "safe hacking".

## Images of changes
This image will be updated every time there will be change:

**Good Luck:**
<details>

https://github.com/ParadiseSS13/Paradise/assets/89580971/4dfd26af-e36e-42c0-88d6-3a3970e91c85

https://github.com/ParadiseSS13/Paradise/assets/89580971/ac264568-87ae-4073-813f-e3930b8cea19
</details>

## Testing
Compiled and it work, but i will do it again after i finish it whole.

## Changelog
:cl: Venuska1117
tweak: Rework Syndicate Research Base
/:cl:

### CODE REVIEW

**Comment by Migratingcocofruit** [comment]
Really like the bomb chamber in the bottom right. Those tend to be a lot less explosives resistant than advertised.  
On a completely different matter, I suggest you put 3 disk boxes in the hydroponics area if it isn't the case already. It makes it more similar to the normal hydroponics setup, and having only 7 disks was a pain besides.

**Comment by Venuska1117** [comment]
> Really like the bomb chamber in the bottom right. Those tend to be a lot less explosives resistant than advertised. On a completely different matter, I suggest you put 3 disk boxes in the hydroponics area if it isn't the case already. It makes it more similar to the normal hydroponics setup, and having only 7 disks was a pain besides.

i think i placed 2 or 3 boxes but i will check tmrw, if i forgot i will add it.

File: code/game/objects/effects/spawners/random_spawners.dm
Comment (Contrabang) [inline code/game/objects/effects/spawners/random_spawners.dm:198]: Please just map these in directly, there is no reason for a 100% chance spawner

Comment (Venuska1117) [reply code/game/objects/effects/spawners/random_spawners.dm:198]: With ones i should map in?

Comment (Venuska1117) [reply code/game/objects/effects/spawners/random_spawners.dm:198]: I dont want to map normal ones because they have only 50% chance to spawn

Comment (Venuska1117) [reply code/game/objects/effects/spawners/random_spawners.dm:198]: ah, nvm i found it

**Comment by Venuska1117** [comment]
Please TM it first before full merge.

**Review by 1080pCat — COMMENTED** [review]
Here is just a few nit picky things

Why are these directional windows plasma windows?
image

Whats the point of having both syndicate, and normal sleepers?
image

The doors to the spawning/cryo area lacks fire locks
image

the toxins setup lacks a disposals inlet/bin
image

The pacman here doesn't have plasma near by, while it isn't really an issue, given the fact this area is meant for newer players to get a feel for doing things, having the fuel across the station may be a little far? Plus, that extra box is begging for a crate with plasma, and a note explaining how to not blow yourself up with the pacman.
image

Whats with this stuff here? Seems like a odd spot to leave a monkey recycler, and a stack of grenade cases, same with the borg charger, theres no real use for that, since you're unable to spawn as a ipc, and you cant get borgs outside of admin spawns.
image

This button is kinda far from the doors? you could probably put one next to each door. you could also add a pair on the inside, i can only imagine people being mean to their fellow researchers and locking them inside the test chamber..

image

the spawners and cryo pods should honestly be facing each other, they look a tad strange not facing each other

image

Overall though this is pretty neat, however there is quite a lack of fire extinguishers, syndicate space OSHA wouldn't be happy!

**Review by Contrabang — CHANGES_REQUESTED** [review]
/obj/item/stack/spacecash/c10000 should not be used. At most it should be 1000 credits, not 10000. If you want to do something that significant, it should be mentioned in the PR description.

**Comment by Venuska1117** [comment]
> Here is just a few nit picky things
> 
> it's a bit long, just a heads up.
> 
> Why are these directional windows plasma windows? image
> 
> Whats the point of having both syndicate, and normal sleepers? image
> 
> The doors to the spawning/cryo area lacks fire locks image
> 
> the toxins setup lacks a disposals inlet/bin image
> 
> The pacman here doesn't have plasma near by, while it isn't really an issue, given the fact this area is meant for newer players to get a feel for doing things, having the fuel across the station may be a little far? Plus, that extra box is begging for a crate with plasma, and a note explaining how to not blow yourself up with the pacman. image
> 
> Whats with this stuff here? Seems like a odd spot to leave a monkey recycler, and a stack of grenade cases, same with the borg charger, theres no real use for that, since you're unable to spawn as a ipc, and you cant get borgs outside of admin spawns. image
> 
> This button is kinda far from the doors? you could probably put one next to each door. you could also add a pair on the inside, i can only imagine people being mean to their fellow researchers and locking them inside the test chamber..
> 
> image
> 
> the spawners and cryo pods should honestly be facing each other, they look a tad strange not facing each other
> 
> image
> 
> Overall though this is pretty neat, however there is quite a lack of fire extinguishers, syndicate space OSHA wouldn't be happy!

i nearly forgot to add fire extingishers, so lets begine with talk:

- The "random grenade" casings are actually metal foam grenades, they are just in case someone decide to blow hole in base.
- Rechargers are for admin spawned cyborgs/maintenance drones, its annoying to walk long distance just to slooooooowly charge.
- About normal sleepers, they are placed for learning, syndicate sleepers and normal sleepers inject different chems.
- The PACMAN generator got placed like this on propuse, its just in case someone dont join syndicate base for more than 30minutes after shift started, at this stage for unknown reason you CAN'T setup thermal generator becayse for some unknown reason pipes require power to work (in this case the ones you interact with), plasma sheets can be found in chemistry and storage.
- Don't worry about "mech" its actually ed209 syndicate version (with look like mech and make semi-mech noise), i just make it look like its actually mech and not scuffed version (with it is)


Other stuff got changed

**Comment by Venuska1117** [comment]
> /obj/item/stack/spacecash/c10000 should not be used. At most it should be 1000 credits, not 10000. If you want to do something that significant, it should be mentioned in the PR description.

tuned down to 2000 credits.

**Comment by meow20** [comment]
The less admins need to bother with whatever happens there, the better.

**Review by 1080pCat — COMMENTED** [review]
I more meant facing the sleepers like this
image

Could probably also 'unfold' these parts a tad, so they're not so outa place?
image

Everything else looks pretty good now though

**Comment by Migratingcocofruit** [comment]
* The PACMAN generator got placed like this on propuse, its just in case someone dont join syndicate base for more than 30minutes after shift started, at this stage for unknown reason you CAN'T setup thermal generator because pipes require power to work (in this case the ones you interact with), plasma sheets can be found in chemistry and storage. (you can do it if you know atmos way, but not everyone know how to replace pipes with different ones)

The current base has an extra powercell near the generator room APC, which works fine. Though it may be a good idea to have APC powercell replacement instructions nearby if you mean to put one there.

**Review by Henri215 — CHANGES_REQUESTED** [review]
1. A blueshield mission briefing here?
image

2. It is not a free version and doesnt make sense for them to have a NANOTRASEN merchandise vendor.

image

3. There are some other vendors that are not free, double check all of them.

4. These should be ```/obj/machinery/economy/vending/toyliberationstation/secured``` because of a recent PR that was merged.

image


5. The players should probably have easy access to the DNA modifier, like it is with all genetics rooms.
image

**Comment by Venuska1117** [comment]
> 1. A blueshield mission briefing here?
>    image
> 2. It is not a free version and doesnt make sense for them to have a NANOTRASEN merchandise vendor.
> 
> image
> 
> 3. There are some other vendors that are not free, double check all of them.
> 4. These should be `/obj/machinery/economy/vending/toyliberationstation/secured` because of a recent PR that was merged.
> 
> image
> 
> 5. The players should probably have easy access to the DNA modifier, like it is with all genetics rooms.
>    image

so this is WHY i had issues with syndie base without finding issue, it was that damn piece of paper.

**Review by 1080pCat — APPROVED** [review]
I don't see any issues, looks pretty good

**Comment by JimKil3** [comment]
Would it be possible to give them a tachyon-doppler array to measure the size of explosions?

**Comment by Venuska1117** [comment]
> Would it be possible to give them a tachyon-doppler array to measure the size of explosions?

Added, and i hope it will not complicate stuff.

File: code/modules/paperwork/paper.dm
Comment (Henri215) [inline code/modules/paperwork/paper.dm:842]:
```suggestion
	info = "P.A.C.M.A.N. are commonly used as 'Emergency' power generators, with its upgraded version being capable of utilizing uranium and plasma sheets to function. Simply anchor on the power cable node, insert the plasma sheet, select the level and turn it ON to generate power, just make sure to not overheat it or it will explode."
```

**Review by Henri215 — COMMENTED** [review]
image

Crittercare costs money and artvend has 2 items that have a cost too.
Oh and claw game costs 5 credits to play too, you could either remove or add a cost free version.. unless its fine in your eyes if they grab some of the money from the vault?

**Review by Henri215 — APPROVED** [review]
LGTM

File: _maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm
Comment (S34NW) [inline _maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm:1]: image
image
These delivery chutes are the two-way ones, you want `/obj/structure/disposaloutlet`
______________

image
This tile snuck in under a door
______________
image
The monkey recycler is linked to the test chamber APC instead of the hallway
______________
image
These floor tiles are wonky
______________
image
These APCs are named but do not need names as they are assigned new names at init
______________
image
This teleporter console is facing a wall
______________
image
I'd add a shutter button on the inside too
______________
image
This defib needs to be the loaded subtype

**Review by S34NW — CHANGES_REQUESTED** [review]

File: _maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm
Comment (Venuska1117) [reply _maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm:1]: defib is unloaded because there is battery nearby, i assume person who made previous outpost wanted to teach people how to insert battery to empty defib.

Comment (S34NW) [inline _maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm:1]: image

**Review by S34NW — CHANGES_REQUESTED** [review]

File: code/modules/ruins/syndicate_space_base.dm
Comment (Sirryan2002) [inline code/modules/ruins/syndicate_space_base.dm:61]: not a big deal, but from a gameplay standpoint, this 100 space credits will pretty much go unused in all cases. Especially when there's already 2k on the new base.

**Review by Sirryan2002 — APPROVED** [review]
code is all good, just waiting on map reviews to finish

File: code/modules/ruins/syndicate_space_base.dm
Comment (Venuska1117) [reply code/modules/ruins/syndicate_space_base.dm:61]: for some reason i cannot figure out how to make claw machine free, so i give syndicate 100 credits to play it.

**Review by S34NW — APPROVED** [review]

**Pull request merged**
